### PR TITLE
developer_guide: mention explicitly the GOPATH dir

### PR DESCRIPTION
**Pull request by hongchaodeng**
[skip ci]

### CODE REVIEW

**Comment by rimusz** [comment]
LGTM

File: doc/dev/developer_guide.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Required tools:
 Build in project root dir:
 
 ```
+( under $GOPATH/src/github.com/coreos/etcd-operator/ )
```
Comment (hasbro17) [inline doc/dev/developer_guide.md]: Maybe put as a bash style comment?
```
# under $GOPATH/src/github.com/coreos/etcd-operator/
```

**Comment by rimusz** [comment]
`hg` should be mentioned too

**Comment by hasbro17** [comment]
LGTM

File: doc/dev/developer_guide.md (hunk 1 above)
Comment (hongchaodeng) [inline doc/dev/developer_guide.md]: This is not really bash script..
It looks weird:
```
# ...
$ ...
```

**Comment by hongchaodeng** [comment]
Added `hg` as required

**Pull request merged**